apps/modernization-ui/src/components/FormInputs/Input.tsx

```diff
@@ -23,6 +23,7 @@ type InputProps = {
     textAreaRef?: RefObject<HTMLTextAreaElement>;
     mask?: string;
     pattern?: string;
+    ariaLabel?: string;
```

Comment:
nit: I think `ariaLabel` is already defined as a prop through `JSX.IntrinsicElements['input']`